File: main/java/ru/carwash/models/Car.java
package ru.carwash.models;

public class Car {

    private String brand;
    private String model;
    private String carNumber;
    private String region;
    private String category;

    public Car(String brand, String model, String carNumber, String region, String category) {
        this.brand = brand;
        this.model = model;
        this.carNumber = carNumber;
        this.region = region;
        this.category = category;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public String getCarNumber() {
        return carNumber;
    }

    public void setCarNumber(String carNumber) {
        this.carNumber = carNumber;
    }

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }
}

File: main/java/ru/carwash/models/Car.kt
package ru.carwash.models

import android.os.Parcel
import android.os.Parcelable

data class Car(
        var brand: String? = null,
        var model: String? = null,
        var carNumber: String? = null,
        var region: String? = null,
        var category: String? = null ) : Parcelable {

    constructor(parcel: Parcel) : this(
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString()) {
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(brand)
        parcel.writeString(model)
        parcel.writeString(carNumber)
        parcel.writeString(region)
        parcel.writeString(category)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Car> {
        override fun createFromParcel(parcel: Parcel): Car {
            return Car(parcel)
        }

        override fun newArray(size: Int): Array<Car?> {
            return arrayOfNulls(size)
        }
    }
}